### DISCUSSION TITLE
Feature request: Progress bars

### DISCUSSION THREAD

**Discussion by ListenerJubatus** [thegeeklab/hugo-geekdoc#580] [Ideas]
Would it be possible to have markup for creating progress bars in pages? We were planning on using them in particular for a documentation page that contains feature roadmap and implementation progress.

**Comment by xoxys** [comment]
Thanks for your request. We have a progress bar short code already implemented in the [hugo-geekblog](https://hugo-geekblog.geekdocs.de/posts/advanced/shortcodes/#progress) theme. What do you think about it? If that works for your use case already, we can easily add it to `hugo-geekdoc` as well.

**Reply by ListenerJubatus** [reply]
Yeah that would be perfect!

**Reply by xoxys** [reply]
Added via https://github.com/thegeeklab/hugo-geekdoc/pull/594
